component-samples/demo/aio/service.yml

```diff
@@ -22,37 +32,36 @@ system-properties:
   
 
 http-server:
-  base_path: app-data
+  base_path: .
   http_port: 8080
   https_port: 8443
   http_schemes:
   - http
   - https
   http_timeout: 20000
   keystore: 
-    path: ssl.p12
-    store-type: PKCS12
-    password: $(ssl_password)
+    path: ./app-data/ssl-host.pem
+    store-type: PEM
+    password: ''
```

Comment:
Is privateKey stored in pem format ?

Reply:
yes, the pem can ---begin certificate--- ..---end certificate ----...---begin private key ---...end private key in the came file.   in openssl you can just cat the cert.pem and key.pem